doc/tools.rst

```diff
@@ -47,6 +47,15 @@ Humongolus
    possible. The code is available for download
    `at GitHub <https://github.com/entone/Humongolus>`_. Tutorials and usage
    examples are also available at GitHub.
+   
+MincePy
+   `MincePy <https://https://mincepy.readthedocs.io/en/latest/>`_ is an 
```

Comment:
The `https://` is repeated so this link won't actually work.

Reply:
Ah, thanks for spotting this!